cromwell.example.backends/cromwell.examples.conf

```diff
@@ -0,0 +1,710 @@
+  #   groupings {
+  #     workflow-option: call-cache-blacklist-group
+  #     concurrency: 10000
+  #     ttl: 2 hours
+  #     size: 1000
+  #   }
+  #
+  #   buckets {
+  #     # Guava cache concurrency.
+  #     concurrency: 10000
+  #     # How long entries in the cache should live from the time of their last access.
+  #     ttl: 20 minutes
+  #     # Maximum number of entries in the cache.
+  #     size: 1000
+  #   }
+  #
+  #   hits {
+  #     # Guava cache concurrency.
+  #     concurrency: 10000
+  #     # How long entries in the cache should live from the time of their last access.
+  #     ttl: 20 minutes
+  #     # Maximum number of entries in the cache.
+  #     size: 100000
+  #   }
+  #
+  # }
+}
+
+# Google configuration
+google {
+
+  #application-name = "cromwell"
+
+  # Default: just application default
+  #auths = [
+
+    # Application default
+    #{
+    #  name = "application-default"
+    #  scheme = "application_default"
+    #},
+
+    # Use a static service account
+    #{
+    #  name = "service-account"
+    #  scheme = "service_account"
+    #  Choose between PEM file and JSON file as a credential format. They're mutually exclusive.
+    #  PEM format:
+    #  service-account-id = "my-service-account"
+    #  pem-file = "/path/to/file.pem"
+    #  JSON format:
+    #  json-file = "/path/to/file.json"
+    #}
+
+    # Use service accounts provided through workflow options
+    #{
+    #   name = "user-service-account"
+    #   scheme = "user_service_account"
+    #}
+  #]
+}
+
+docker {
+  hash-lookup {
+    # Set this to match your available quota against the Google Container Engine API
+    #gcr-api-queries-per-100-seconds = 1000
+
+    # Time in minutes before an entry expires from the docker hashes cache and needs to be fetched again
+    #cache-entry-ttl = "20 minutes"
+
+    # Maximum number of elements to be kept in the cache. If the limit is reached, old elements will be removed from the cache
+    #cache-size = 200
+
+    # How should docker hashes be looked up. Possible values are "local" and "remote"
+    # "local": Lookup hashes on the local docker daemon using the cli
+    # "remote": Lookup hashes on docker hub, gcr, gar, quay
+    #method = "remote"
+  }
+}
+
+engine {
+  # This instructs the engine which filesystems are at its disposal to perform any IO operation that it might need.
+  # For instance, WDL variables declared at the Workflow level will be evaluated using the filesystems declared here.
+  # If you intend to be able to run workflows with this kind of declarations:
+  # workflow {
+  #    String str = read_string("gs://bucket/my-file.txt")
+  # }
+  # You will need to provide the engine with a gcs filesystem
+  # Note that the default filesystem (local) is always available.
+  filesystems {
+    #  gcs {
+    #    auth = "application-default"
+    #    # Google project which will be billed for the requests
+    #    project = "google-billing-project"
+    #  }
+    #  oss {
```

Comment:
We may want to replace or remove this filesystem from the example since we no longer support it.